src/config/consts.js

```diff
@@ -50,8 +50,8 @@ export const consts = {
     settingsIcon: 'Einstellungen (Taste)',
     settingsHint: 'Anpassen des Layouts von',
     settingsIconHint: 'Zu den Einstellungen wechseln',
-    settingsBookmarksIcon: 'Einstellungen und Lesezeichen (Taste)',
-    settingsBookmarksHint: 'Zu den Einstellungen und Lesezeichen wechseln',
+    settingsBookmarksIcon: 'Lesezeichen (Taste)',
+    settingsBookmarksHint: 'Zu den Lesezeichen wechseln',
```

Comment:
we do not have settings here anymore. please also rename the key to make it fit.

Reply:
Updated 👍

https://github.com/smart-village-solutions/smart-village-app-app/commit/76c9de881368178dcf3a183b4ff6c2f9aad18d0a